IFC4x3/Properties/a/AvgSpacingOfTreatment_3IwKbHD4H2mxQff$jGfKO3/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="AvgSpacingOfTreatment_3IwKbHD4H2mxQffjGfKO3" Name="AvgSpacingOfTreatment" UniqueId="d2e94951-3444-42c3-b6a9-a7fb50a54603" PrimaryDataType="IfcPositiveLengthMeasure" />
```

Comment:
Propose renaming to "AvgTreatmentSpacing".

Reply:
Fixed in [fa34e3f](https://github.com/bSI-InfraRoom/IFC-Specification/pull/676/commits/fa34e3f56f97c85fda90a91839411efbd6712470)